### DISCUSSION TITLE
transient hydraulic loads

### DISCUSSION THREAD

**Discussion by mrs11liu** [Anura3D/Anura3D_OpenSource#19] [General]
Hey All,

I am following tutorial 11 from the manual.  When running the first stage of gravity loading, the program failed. I don't know what went wrong.

![image](https://user-images.githubusercontent.com/54770340/178155860-62b3f833-586d-4b01-bddc-da517d599800.png)
After running, the generated files are as follows.
![image](https://user-images.githubusercontent.com/54770340/178155901-8b65dbcf-7ffa-45ac-8bf5-45a60e0660e4.png)

**Comment by ayerrocolom** [comment]
This error might be due to several reasons, but generally it means that you are using memory you're not allowed to (e.g., trying to accessing an uninitialized variable). First, I would make sure that you didn't miss any step from the tutorial manual, and secondly, I would try to run the code with the debugging tool in Visual Studio to identify the problem.
